resources/datadog-integrations/project.rb.erb

```diff
@@ -43,8 +43,11 @@ end
 dependency 'preparation'  # creates required build directories
 dependency 'dd-check-<%= name %>-software'
 
+# Checks should require the datadog agent
+runtime_dependency 'datadog-agent'
+
 extends_packages('datadog-agent', 'datadog')
 
-extra_package_file '/etc/dd-agent/conf.d/<%= name %>.yaml.example'
+extra_package_file '/etc/dd-agent/conf.d/examples/<%= name %>.yaml.example'
```

Comment:
@degemer @irabinovitch the checks that are shipped with the agent will continue to have all of their `conf.yaml.example` files reside in `/etc/dd-agent/conf.d`. The only ones that will have the config in `/etc/dd-agent/conf.d/examples` will be the ones installed via the SDK. Will any of these be the default ones?

Even if they are, we cannot overwrite the default config we already have, so we'll have to place it elsewhere. This leaves us with two options:

1. Do we drop them in examples, with the rest of them?

1. Do we add a new folder for them?

I think leaving them with the rest of the sdk ones makes the most sense. Doing it the other way would add another layer for the user to comprehend.